.github/workflows/deploy_docs.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         run: poetry install --no-root
 
       - name: Deploy website
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

Comment:
This action was trying to deploy docs in the PR, which isn't ideal. This just makes sure we don't do the actual docs deploy step unless this is a real push to main.